packages/widget_toolkit_pin/lib/src/lib_pin_code_with_biometrics/data_source/pin_biometrics_auth_data_source.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:local_auth/local_auth.dart';
+import 'package:widget_toolkit_biometrics/widget_toolkit_biometrics.dart';
+
+import '../models/biometrics_authentication_type.dart';
+
+class PinBiometricsAuthDataSource extends BiometricsAuthDataSource {
+  PinBiometricsAuthDataSource({required super.localAuthentication});
+
+  Future<List<BiometricsAuthType>> get availableBiometrics =>
+      localAuthentication
+          .getAvailableBiometrics()
+          .then((list) => list.map((e) => mapBiometric(e)).toList());
+
+  BiometricsAuthType mapBiometric(BiometricType type) {
+    switch (type) {
+      case BiometricType.face:
+        return BiometricsAuthType.face;
+      case BiometricType.fingerprint:
+        return BiometricsAuthType.fingerprint;
+      case BiometricType.iris:
+        return BiometricsAuthType.iris;
+      case BiometricType.strong:
+        return BiometricsAuthType.face;
+      case BiometricType.weak:
+        return BiometricsAuthType.face;
```

Comment:
Since there are several cases which return the same response what you can do is group the same cases one above the other and allow them to pass through each other eventually returning the value. That is, in code it would look like this:

```dart
switch (type) {
  case BiometricType.fingerprint:
    return BiometricsAuthType.fingerprint;
  case BiometricType.iris:
    return BiometricsAuthType.iris;
  case BiometricType.strong:
  case BiometricType.weak:
  case BiometricType.face:
    return BiometricsAuthType.face;
}
```

Reply:
The change was applied.